packages/blade/src/components/Box/BaseBox/baseBoxStyles.ts

```diff
@@ -221,6 +221,9 @@ const getAllProps = (
     borderRight: getResponsiveValue(props.borderRight, breakpoint),
     borderBottom: getResponsiveValue(props.borderBottom, breakpoint),
     borderLeft: getResponsiveValue(props.borderLeft, breakpoint),
+    touchAction: props.touchAction,
+    userSelect: props.userSelect,
+    opacity: props.opacity,
```

Comment:
```suggestion
    touchAction: getResponsiveValue(props.touchAction, breakpoint),
    userSelect:  getResponsiveValue(props.userSelect, breakpoint),,
    opacity:  getResponsiveValue(props.opacity, breakpoint),,
```

this handles the `{ base: , xs: }` object

Reply:
Noice thanks!